pkg/storage/installation_store.go

```diff
@@ -63,14 +63,16 @@ func (s InstallationStore) Initialize(ctx context.Context) error {
 	return span.Error(err)
 }
 
-func (s InstallationStore) ListInstallations(ctx context.Context, namespace string, name string, labels map[string]string) ([]Installation, error) {
+func (s InstallationStore) ListInstallations(ctx context.Context, namespace string, name string, labels map[string]string, skip int64, limit int64) ([]Installation, error) {
```

Comment:
@carolynvs Actually been thinking about that as well! Will add it on the next commits.

Also, what do you think of making the `--skip` and `--limit` options available to `porter credentials list` and `porter parameters list` commands as well? If yes, I can also do the process of changing the function argument to option struct for those two as well.

Reply:
@carolynvs should've been ready for another round of review! I made the option struct a general one (`ListOptions`) so that it can also be used for ListParameters and ListCredentialSets but looking forward to hearing your thoughts!